src/WorldIDIdentityManagerImplV1.sol

```diff
@@ -82,6 +82,9 @@ contract WorldIDIdentityManagerImplV1 is OwnableUpgradeable, UUPSUpgradeable, IW
     /// @notice The verifier instance needed for operating within the semaphore protocol.
     SemaphoreVerifier private semaphoreVerifier;
 
+    /// @notice The interface of the bridge contract from L1 to supported target chains.
+    address public stateBridgeProxy;
```

Comment:
Maybe I'm seeing something wrong, but this still shows as `public` to me, and without a leading underscore.

![image](https://user-images.githubusercontent.com/5780639/214446332-216b0a75-6c9a-4f66-8c28-7a0e6f36c6eb.png)

Reply:
isn't that how it should be? Why should it have an underscore? underscores to me are generally used to indicate that something is a parameter to avoid name clashes. You also use it for internal variables which I guess is ok as well, but then I'd have a name collision in the `initialize` function

Reply:
I made the variable internal